Revealed/Design Components/Color.swift

```diff
@@ -0,0 +1,14 @@
+//
+import Foundation
+//  Color.swift
+//  Revealed
+//
+//  Created by Hong on 26/3/20.
+//  Copyright © 2020 Pointwelve. All rights reserved.
+//
+import SwiftUI
+
+extension Color {
+  static let charcoalGrey60 = Color(red: 60.0 / 255.0, green: 60.0 / 255.0, blue: 67.0 / 255.0, opacity: 0.6)
```

Comment:
we should use system color.

Reply:
hmm, zeplins doesn't show system color leh. We should create a styleguide right.

Reply:
ya it wont show, thats why i have to create a style guide, for now can just leave it, once i finish we can clean up.